src/fuel_fab.h

```diff
@@ -199,9 +199,10 @@ class FuelFab : public cyclus::Facility {
   #pragma cyclus var { \
     "doc": "Top-up material stream request preference.", \
     "uilabel": "Top-up Stream Preference", \
-    "default": 0, \
+    "default": 0.01, \
```

Comment:
We should probably just make this 1.  Could you make that change here? - 0.01 just stands out like an eyesore and would likely surprise users.